tests/Unit/CrudPanel/CrudPanelColumnsTest.php

```diff
@@ -154,7 +154,9 @@ public function testAddColumnsAsArray()
 
     public function testAddColumnNotArray()
     {
-        $this->expectException(\ErrorException::class);
+        $this->expectException(PHP_MAJOR_VERSION < 8 ? \ErrorException::class : \TypeError::class);
```

Comment:
```suggestion
        $this->expectException(PHP_MAJOR_VERSION == 7 ? \ErrorException::class : \TypeError::class);
```

But I'd still do `==` instead of `<`. Easier to read in my opinion, and the same thing since we only ever supported PHP 7 and 8 in this version.